discovery-infra/test_infra/logs_utils.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import os
+import shutil
+import tarfile
+import time
+from tempfile import TemporaryDirectory
+
+from test_infra.consts import NUMBER_OF_MASTERS
+
+
+def verify_logs_uploaded_and_delete(cluster_tar_path, expected_min_log_num, installation_success):
```

Comment:
I dont understand why u need it in this context. How does it help?

Reply:
I don't need it. These functions are also used in the e2e logs tests.
Anyway, this function got deleted.